action/execution_test.go

```diff
@@ -36,8 +36,6 @@ func TestExecutionSignVerify(t *testing.T) {
 	elp, ok := eb.(*envelope)
 	require.True(ok)
 
-	require.Equal("0801100218a08d0622023130622f0a0231301229696f3172633264326465377274757563616c7371763464396e673068323937743633773777766c7068", hex.EncodeToString(elp.serialize()))
```

Comment:
suggest to keep `serialize()` and related tests, b/c they help verify protobuf backward compatibility